dynamiqs/rand.py

```diff
@@ -0,0 +1,34 @@
+from jax import numpy as jnp, Array
+import jax
+
+from dynamiqs.utils import dag  # todo: clean this dependency
+
+
+def complex_matrix(dims: tuple[int, ...], key=None) -> Array:
+    key = key if key is not None else jax.random.PRNGKey(0)
+    key1, key2 = jax.random.split(key, 2)
+    mat = jax.random.normal(key1, dims) + 1j * jax.random.normal(key2, dims)
+    return mat
+
+
+def hermitian(n: int, key=None) -> Array:
+    key = key if key is not None else jax.random.PRNGKey(0)
+    key1, key2 = jax.random.split(key, 2)
+    mat = 2 * jax.random.uniform(key1, (n, n)) - 1
+    mat = mat + 1j * (2 * jax.random.uniform(key2, (n, n)) - 1)
+    mat = 0.5 * (mat + dag(mat))
```

Comment:
```suggestion
    mat = complex_matrix((n, n), key=key)
    return 0.5 * (mat + dag(mat))
```
Why not something like this?

Reply:
Honestly I copy-pasted qutip's code, but this may be simpler indeed